tests/test_cli.py

```diff
@@ -176,12 +187,67 @@ def test_push_tracked(runner):
         assert push_result.output.strip() == expected_output
 
 
+def test_push_multi(runner):
+    with runner.isolated_filesystem():
+        query_49741_before = setup_tracked_query(runner, '49741', '49741.sql',
+                                                 response_49741_content)
+        query_62375_before = setup_tracked_query(runner, '62375', '62375.sql',
+                                                 response_62375_content)
+
+        # Overwrite with a new query string
+        query_49741_after = update_tracked_query('49741.sql', query_49741_before)
+        query_62375_after = update_tracked_query('62375.sql', query_62375_before)
+
+        # Now push the result
+        with HTTMock(push_response):
+            push_result = runner.invoke(cli.cli, [
+                "push",
+                "49741.sql",
+                "62375.sql"
+            ])
+
+        m = hashlib.md5(query_62375_after.encode("utf-8"))
+        expected_output_62375 = "Query ID 62375 updated with content from 62375.sql " + \
+                                "(md5 {})".format(m.hexdigest())
+        m = hashlib.md5(query_49741_after.encode("utf-8"))
+        expected_output_49741 = "Query ID 49741 updated with content from 49741.sql " + \
+                                "(md5 {})".format(m.hexdigest())
+
+        assert push_result.output.strip() == expected_output_49741 + "\n" + expected_output_62375
+
+def test_push_all(runner):
+    with runner.isolated_filesystem():
+        query_49741_before = setup_tracked_query(runner, '49741', '49741.sql',
+                                                 response_49741_content)
+        query_62375_before = setup_tracked_query(runner, '62375', '62375.sql',
+                                                 response_62375_content)
+
+        # Overwrite with a new query string
+        query_49741_after = update_tracked_query('49741.sql', query_49741_before)
+        query_62375_after = update_tracked_query('62375.sql', query_62375_before)
+
+        # Now push the result
+        with HTTMock(push_response):
+            push_result = runner.invoke(cli.cli, [
+                "push"
+            ])
+
+        m = hashlib.md5(query_62375_after.encode("utf-8"))
+        expected_output_62375 = "Query ID 62375 updated with content from 62375.sql " + \
+                                "(md5 {})".format(m.hexdigest())
+        m = hashlib.md5(query_49741_after.encode("utf-8"))
+        expected_output_49741 = "Query ID 49741 updated with content from 49741.sql " + \
+                                "(md5 {})".format(m.hexdigest())
+
+        assert push_result.output.strip() == expected_output_49741 + "\n" + expected_output_62375
```

Comment:
Looks like this test is fragile. The CI for python 2.7 handles 62375 before handling 49741 so these strings don't match. Instead, we can either: 

1. Exclusively verify that each MD5 hash is included in the output
2. Make this more flexible by splitting the actual output on newlines and doing a `set` equals

Reply:
I'll go with the set() approach, thanks for the suggestion